magmi/engines/magmi_productimportengine.php

```diff
@@ -106,12 +106,23 @@ public function getEngineInfo()
      */
     public function initProdType()
     {
-        $tname = $this->tablename("eav_entity_type");
-        $this->prod_etype = $this->selectone("SELECT entity_type_id FROM $tname WHERE entity_type_code=?",
-            "catalog_product", "entity_type_id");
+        $this->fetchProdEType();
         $this->default_asid = $this->getAttributeSetId('Default');
     }
 
+    /**
+     * Fetches the entity type for "catalog_product" and stores it to $this->prod_etype (if not already done).
+     */
```

Comment:
Instead of adding a comment why not give method a descriptive name (e.g. `setProductEntityType` or `initializeProductEntityType`)?